contracts/contracts/recipientRegistry/OptimisticRecipientRegistry.sol

```diff
@@ -129,7 +129,6 @@ contract OptimisticRecipientRegistry is Ownable, BaseRecipientRegistry {
     require(recipients[_recipientId].index != 0, 'RecipientRegistry: Recipient is not in the registry');
     require(recipients[_recipientId].removedAt == 0, 'RecipientRegistry: Recipient already removed');
     require(requests[_recipientId].submissionTime == 0, 'RecipientRegistry: Request already submitted');
-    require(msg.value == baseDeposit, 'RecipientRegistry: Incorrect deposit amount');
```

Comment:
Double-check this. I'm not sure if we want to remove this because anyone could still keep spamming without using the UI.

Reply:
This issue should be covered in a next PR when we tackle https://www.notion.so/efdn/Restrict-project-removals-to-admin-only-82baf36ae696473a8228f899a98a51eb
Since the removal will be admin only, we are not going to have spamming issues anymore.